pkg/config/vars.go

```diff
@@ -94,9 +94,19 @@ var configVars = map[string]configVar{
 		Default: false,
 		CLIFlag: cli.BoolFlag{
 			Name:   "disable-metrics",
-			Usage:  "disable Prometheus metrics",
+			Usage:  "(deprecated)disable Prometheus metrics",
```

Comment:
can we add a space between `(deprecated)` and `disable Prometheus metrics` ?

Reply:
Sure, I will into this today.